testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -794,7 +794,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
 @uyuni
 @proxy
   Scenario: Add Uyuni Leap 15.5 Proxy, including Uyuni Client Tools
-    When I use spacewalk-common-channel to add all "leap15.5" channels with arch "x86_64"
+    When I use spacewalk-common-channel to add all "uyuni-proxy" channels with arch "x86_64"
```

Comment:
I don't get it here, why in `Uyuni Leap 15.5 Proxy` we are adding micro5.5 channels instead?

Reply:
Good catch, the scenario name is not correct.